script.js

```diff
@@ -16,9 +16,9 @@ function fillTableFromDict(summaryTableData) {
 }
 
 function getEUSummary(allCountries) {
-  var EUCodes = Object.keys(EUCountries).map(key => EUCountries[key]);
-  EUCodes = EUCodes.concat(Object.keys(EEACountries).map(key => EEACountries[key]));
-  EUCodes = EUCodes.concat(Object.keys(UKCountries).map(key => UKCountries[key]));
+  var EUCodes = Object.keys(EUCountries);
+  EUCodes = EUCodes.concat(Object.keys(EEACountries));
+  EUCodes = EUCodes.concat(Object.keys(UKCountries));
```

Comment:
you can get eucodes in a bit different way (in one string :))
`const EUCodes = Object.keys(Object.assign(EUCountries, EEACountries, UKCountries));`